src/custom-element.js

```diff
@@ -381,6 +381,9 @@ export function createAmpElementProto(win, name, opt_implementationClass) {
      * @private {boolean|undefined}
      */
     this.isInTemplate_ = undefined;
+
+    /** @private {boolean} */
+    this.hadLoadingError_ = false;
```

Comment:
I'll use the existing `this.layoutCount_` value, I agree it's definitely better, and terminate loading animation for element should be the same process despite loading status.